examples/transfer_subscribe.rs

```diff
@@ -37,11 +37,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let dest = AccountKeyring::Bob.to_account_id().into();
 
     let client = ClientBuilder::<DefaultNodeRuntime>::new().build().await?;
-    let sub = client.subscribe_events().await?;
-    let mut decoder = EventsDecoder::<DefaultNodeRuntime>::new(client.metadata().clone());
-    decoder.with_balances();
```

Comment:
@dvc94ch is there a way to form this call within `event_subscription` method I added below from the `E: Event<T>` associated type? 

Like this, but I can't use the proc_macro2 or quote....
```rust
let decoder_with_module: proc_macro2::TokenStream = quote!(quote::format_ident!(
        "decoder.with_{}()",
        E::MODULE.to_string().to_snake_case()
    )).into();
    decoder_with_module;
```